meteor/client/ui/Settings/StudioSettings.tsx

```diff
@@ -1206,6 +1207,9 @@ class StudioMappings extends React.Component<Translated<IStudioMappingsProps>, I
 
 		return (
 			_.map(this.props.studioInstallation.mappings, (mapping: Mapping , layerId: string) => {
+				// If an internal mapping, then hide it
+				if (mapping.internal) return <React.Fragment key={layerId}></React.Fragment>
```

Comment:
This probably performs better, but it would be cleaner to just filter the `.mappings` array

Reply:
Mappings is an object not an array, so _.filter mangles the result so that we no longer have the layerId

Reply:
OK, I guess we can leave this as-is.